pymilvus/client/interceptor.py

```diff
@@ -13,16 +13,17 @@
 # limitations under the License.
 """Base class for interceptors that operate on all RPC types."""
 
+import time
 from typing import Any, Callable, List, NamedTuple, Optional, Tuple
 
 import grpc
 
 
 def _api_level_md(**kwargs) -> Optional[List[Tuple]]:
-    metadata = None
+    metadata = []
+    metadata.append(("client-request-unixmsec", str(time.time_ns())))
```

Comment:
IMHO, time.time_ns() returns nano seconds, this header value shall be in millisecond